README.md

```diff
@@ -411,6 +411,25 @@ The following table lists events for this API.
 |`event.stats` | The `event.stats` event is emitted when `librdkafka` reports stats (if you opted in). |
 |`event.throttle` | The `event.throttle` event is emitted when `librdkafka` reports throttling.|
 
+## Reading current offsets from the broker for a topic
+
+Some times you find yourself in the situation where you need to know the latest (and earliest) offset for one of your topics. Connected producers and consumers both allow you to query for these through `queryWaterMarkOffsets` like follows:
+
+```js
+const timeout = 5000, partition = 0;
```

Comment:
Please change these `const`s to `var`s, just to be consistent among the rest of the README.